adminservice/src/main/java/nu/yona/server/AdminServiceApplication.java

```diff
@@ -38,8 +40,11 @@
 @SpringBootApplication
 @EnableSwagger2
 @EnableHypermediaSupport(type = HypermediaType.HAL)
+@EnableCaching
 public class AdminServiceApplication
 {
+	@Autowired
+	CacheManager cacheManager;
```

Comment:
Scrolling further down, I see you're defining the bean on ``CoreConfiguration``, but not instantiating it there. I propose to centralize the instantiation of the cache manager in the core component layer, so everyone can rely in its availability.